### DISCUSSION TITLE
예약신청 페이지에 개인정보수집동의여부 문의

### DISCUSSION THREAD

**Discussion by YounghuSong** [changdoc/naver-vaccine-macro#140] [Q&A]
안녕하세요. 제작해주신 덕분에 정말 감사히 사용하고 있습니다.

궁금한 부분이 있어 문의드립니다.

기존에는 예약신청페이지에서 개인정보수집여부가 있었다가 현재는 사라진 것 같은데요.

그 부분이 사라지면서 예약실패가 뜨는 상황이 발생하는 것 같습니다.

예약실패가 뜨는 원인이 맞는지 확인 한 번 해주실 수 있으신지요 ?

![image](https://user-images.githubusercontent.com/77137340/127425947-baaa77f5-92d0-4bd0-bff7-78f40b43ad6b.png)

**Comment by changdoc** [comment]
예약 시도를 한번 하게 되면 제공 동의를 한것으로 간주되어 페이지에 노출이 안되게 되어 있는것으로 파악됩니다.

동의 여부는 예약 성공/실패에 영향을 주지 않는것으로 알고 있습니다.
